source/Calamari.Tests/Fixtures/Integration/FileSystem/CalamariPhysicalFileSytemFixture.cs

```diff
@@ -91,5 +91,57 @@ string CreateFile(params string[] relativePath)
             return filename;
         }
 
+        [Test]
+        [TestCase(@"**/*.txt", "f1.txt", 2)]
+        [TestCase(@"**/*.txt", "r.txt", 2)]
+        [TestCase(@"*.txt", "r.txt")]
+        [TestCase(@"**/*.config", "root.config", 5)]
+        [TestCase(@"*.config", "root.config")]
+        [TestCase(@"Config/*.config", "c.config")]
+        [TestCase(@"Config/Feature1/*.config", "f1-a.config", 2)]
+        [TestCase(@"Config/Feature1/*.config", "f1-b.config", 2)]
+        [TestCase(@"Config/Feature2/*.config", "f2.config")]
+        public void GlobTestMutiple(string pattern, string expectedFileMatchName, int expectedQty = 1)
+        {
+            var realFileSystem = CalamariPhysicalFileSystem.GetPhysicalFileSystem();
+            var rootPath = realFileSystem.CreateTemporaryDirectory();
+            var content = "file-content" + Environment.NewLine;
+
+            if (CalamariEnvironment.IsRunningOnWindows)
+            {
+                pattern = pattern.Replace(@"/", @"\");
+            }
+
+            try
+            {
+                var configPath = Path.Combine(rootPath, "Config");
+
+                realFileSystem.CreateDirectory(configPath);
```

Comment:
Why not just use `Directory.CreateDirectory`, then you wouldn't need to change CalamariPhysicalFileSystem